scripts/mint.ts

```diff
@@ -156,23 +160,23 @@ async function mintForRange(tokenIdRange: string, recipient: string) {
   console.log('Tx hash: ', result.transactionHash);
 }
 
-const args = process.argv.slice(2);
-if (args.length == 0) {
-  console.log('No arguments provided, need --to or --for');
-} else if (args.length == 1 && args[0] == '--test-whitelist') {
-  test_whitelist();
-} else if (args.length == 2 && args[0] == '--to') {
-  mintTo(args[1]);
-} else if (args.length == 4 && args[0] == '--to') {
-  if (args[2] == '--batch') {
-    batchMint(args[1], +args[3]);
-  } else {
-    console.log('Invalid arguments');
-  }
-} else if (args.length == 3 && args[0] == '--for') {
-  mintFor(args[1], args[2]);
-} else if (args.length == 3 && args[0] == '--range') {
-  mintForRange(args[1], args[2]);
-} else {
-  console.log('Invalid arguments');
-}
+// const args = process.argv.slice(2);
+// if (args.length == 0) {
+//   console.log('No arguments provided, need --to or --for');
+// } else if (args.length == 1 && args[0] == '--test-whitelist') {
+//   test_whitelist();
+// } else if (args.length == 2 && args[0] == '--to') {
+//   mintTo(args[1]);
+// } else if (args.length == 4 && args[0] == '--to') {
+//   if (args[2] == '--batch') {
+//     batchMint(args[1], +args[3]);
+//   } else {
+//     console.log('Invalid arguments');
+//   }
+// } else if (args.length == 3 && args[0] == '--for') {
+//   mintFor(args[1], args[2]);
+// } else if (args.length == 3 && args[0] == '--range') {
+//   mintForRange(args[1], args[2]);
+// } else {
+//   console.log('Invalid arguments');
+// }
```

Comment:
why is this commented?

Reply:
because `mintTo` is imported from `mint.ts`, the args are processed twice. commenting avoids that but not sure what the real fix is